### DISCUSSION TITLE
Received Token from metamask ETH to SOL phantom wallet, but as NFT

### DISCUSSION THREAD

**Discussion by BLACKoin** [wormhole-foundation/wormhole#532] [Problems / bugs] [answered]
**has same issue:** @leoluk @maromandarino

ETH Txn: 0x9560685ee29f28e76b522c9338f080c27f67d1e184c0348ba76ac52f75dbf253
Sollana wallet address: PQJVRCJ7rtpvZXFXEWXCW9i91hN8p4hxoqeuHjiCrDP
sent to: AoyQUXh5GSnAmfvUTGTfLvRrazYCegpistkLL3MArwoj
 
![image](https://user-images.githubusercontent.com/92128631/136478939-3b49ae34-f6fa-4284-b757-7564a1110060.png)

![image](https://user-images.githubusercontent.com/92128631/136478987-0e57e299-f560-442d-ba2a-ef30d44d3343.png)

**Comment by leoluk** [comment, marked as answer]
UI bug in Phantom which causes your tokens to appear as NFTs. Should be resolved by now.
